### PR TITLE
Add Indonesia

### PR DESCRIPTION
**Pull request by ikopanji**

### CODE REVIEW

File: config/regions.yaml [hunk 1]
```diff
@@ -79,6 +79,10 @@ regions:
     name: Hong Kong
     flag: 🇭🇰
     osmcha.id: ddd87c1a-717b-4d30-815a-507ed8d57a3c
+  indonesia:
+    name: Indonesia
+    flag: id
```
Comment (jbelien) [inline config/regions.yaml]: Must be an emoji!

```suggestion
    flag: 🇮🇩
```

**Review by jbelien — CHANGES_REQUESTED** [review]
Hello @ikopanji ,

Don't forget to add the GeoJSON file of your region!
Check https://github.com/osmbe/osm-welcome-tool/blob/2.x/CONTRIBUTING.md#add-a-new-region

File: config/regions.yaml (hunk 1 above)
Comment (ikopanji) [reply config/regions.yaml]: Dear @jbelien,
I've already uploaded the area for Indonesia and updated the code to "ID" in accordance with your suggestion.

Comment (jbelien) [reply config/regions.yaml]: I've merged #672 into this PR and cleaned your GeoJSON file.

⚠️ The `flag` property MUST be an emoji (not a text, not `id`, not `ID`)! See my suggested change!

Comment (ikopanji) [reply config/regions.yaml]: Oops, already fixed.

**Review by jbelien — APPROVED** [review]

**Comment by jbelien** [comment]
🚀 Deployed: https://welcome.osm.be/indonesia/list
(process is running)

**Pull request merged**